test/ShapeCrawler.Tests.Unit/ShapeCollectionTests.cs

```diff
@@ -526,26 +530,27 @@ public void AddPicture_adds_svg_picture_no_width_height_tags()
         picture.Width.Should().Be(280);
         pres.Validate();
     }
-
+    
+    //Test that the png image is created with transparent background
     [Test]
-    public void AddPicture_adds_svg_picture_no_dimensions()
```

Comment:
Are we abandoning this test `AddPicture_adds_svg_picture_no_dimensions`?

@jcoliz, I've run this test `AddPicture_adds_svg_picture_no_dimensions` on the latest changes of actual PR. It fails: The actual values:

```C#
picture.Height == 90
picture.Width == 276
```

Reply:
We had this failure earlier on. The new package doesn't calculate the size perfectly. However, due to how PowerPoint handles SVGs, the reality is that as soon as you save the PowerPoint, it gets recalculated incorrectly anyway. Hence, the suggestion of stating dimensionless SVGs produce undefined behaviour.

Reply:
Yup. This test is not helpful. It's true, and it works, but the outcome is not what we want.

As an example, imagine a 200x100 solid black image. It will be correctly displayed as 200x100 in PowerPoint. However, the black area is scaled down to 20x5, and the remaining pixels are blank.

You can try by saving the result of this test to disk and then load in PowerPoint.